FSW/src/headers/config.hpp

```diff
@@ -69,13 +80,51 @@ static Event saveBufferEvent;
 static TimedEvent sunriseTimerEvent(WINDOW_LENGTH_MSEC);
 static TimedEvent sweepTimeoutEvent(SWEEP_TIMEOUT_MSEC);
 
+// FUTURE TEAMS: this event is invoked when the ADCS should switch its sweep direction
+//   so link your ADCS module with this event to tell it when to switch direction 
+//      look at checkSweepChange() in timing.cpp for more info
+static Event sweepDirectionChangeEvent; 
+
+

```

Comment:
This event got moved into the `timingClass.hpp` header, it shouldn't be in this file anymore. I wrote a note to circle back and look at the file once you merge it into main, if this shows back up then we will have to remove it, but the merge may take care of that.